kofta/src/app/modules/room-chat/EmoteData.ts

```diff
@@ -368,94 +362,34 @@ export const customEmojis = [
     imageUrl: "/emotes/feelsgoodman.png",
   },
   {
-    name: "GitHub",
-    short_names: ["GitHub"],
-    keywords: ["app", "git", "github"],
-    imageUrl: "/emotes/github.png",
-  },
-  {
-    name: "Elixir",
-    short_names: ["Elixir"],
-    keywords: ["app", "elixir"],
-    imageUrl: "/emotes/elixir.png",
-  },
-  {
-    name: "NextJS",
-    short_names: ["NextJS"],
-    keywords: ["app", "next", "js"],
-    imageUrl: "/emotes/nextjs.png",
-  },
-  {
-    name: "React",
-    short_names: ["React"],
-    keywords: ["app", "react", "js"],
-    imageUrl: "/emotes/react.png",
-  },
-  {
-    name: "Electron",
-    short_names: ["Electron"],
-    keywords: ["app", "electron"],
-    imageUrl: "/emotes/electron.png",
-  },
-  {
-    name: "BeardGuyR",
-    short_names: ["BeardGuyR"],
-    keywords: ["meme", "beard", "guy"],
-    imageUrl: "/emotes/beardguyright.png",
-  },
-  {
-    name: "BeardGuyL",
-    short_names: ["BeardGuyL"],
-    keywords: ["meme", "beard", "guy"],
-    imageUrl: "/emotes/beardguyleft.png",
-  },
-  {
-    name: "MemeMan",
-    short_names: ["MemeMan"],
-    keywords: ["meme", "guy"],
-    imageUrl: "/emotes/mememan.png",
-  },
-  {
-    name: "NPC",
-    short_names: ["NPC"],
-    keywords: ["meme", "npc", "guy"],
-    imageUrl: "/emotes/npc.png",
-  },
-  {
-    name: "SmudgeCat",
-    short_names: ["SMudgeCat"],
-    keywords: ["meme", "cat"],
-    imageUrl: "/emotes/smudgecat.png",
-  },
-  {
-    name: "Stonks",
-    short_names: ["Stonks"],
-    keywords: ["meme", "crypto", "guy"],
-    imageUrl: "/emotes/stonks.png",
+    name: "WidePeepoPussy",
+    short_names: ["WidePeepoPussy"],
+    keywords: ["WidePeepoPussy", "peepo", "wide"],
+    imageUrl: "/emotes/widepeepoPussy.png",
   },
   {
-    name: "Stinks",
-    short_names: ["Stinks"],
-    keywords: ["meme", "crypto", "guy"],
-    imageUrl: "/emotes/stinks.png",
+    name: "OmegaLuL",
+    short_names: ["OmegaLuL"],
+    keywords: ["OmegaLuL", "LuL", "Omega"],
+    imageUrl: "/emotes/omegaLul.png",
   },
   {
-    name: "IWMD",
-    short_names: ["IWMD"],
-    keywords: ["meme", "dude", "wednesday", "frog"],
-    imageUrl: "/emotes/wednesday.png",
+    name: "CrabPls",
+    short_names: ["CrabPls"],
+    keywords: ["Crab", "Pls"],
+    imageUrl: "/emotes/crabPls.gif",
   },
   {
-    name: "PreSponge",
-    short_names: ["PreSponge"],
-    keywords: ["meme", "spongebob"],
-    imageUrl: "/emotes/presponge.png",
+    name: "RareParrot",
+    short_names: ["RareParrot"],
+    keywords: ["Rare", "Parrot", "party"],
+    imageUrl: "/emotes/rareParrot.gif",
   },
   {
-    name: "sPonGeBoB",
-    short_names: ["sPonGeBoB"],
-    keywords: ["meme", "spongebob"],
-    imageUrl: "/emotes/sPonGeBoB.png",
```

Comment:
The two that were removed here shouldn't be removed either.

Reply:
Nothingf from the code should be removed